CMakeLists.txt

```diff
@@ -498,7 +499,7 @@ ADD_CUSTOM_TARGET(dist
 			COMMAND make clean
 			COMMAND rm -rf "${TMP}"
 			COMMAND mkdir -p "${TMP}"
-			COMMAND cp AUTHORS build_mingw32 build_mingw64 CMakeLists.txt configure COPYING INSTALL lmms.1 lmms.rc.in lmms.spec.in lmmsconfig.h.in lmmsversion.h.in README TODO "${TMP}"
+			COMMAND cp AUTHORS build_mingw32 build_mingw64 CMakeLists.txt configure COPYING INSTALL README.md TODO "${TMP}"
```

Comment:
@Wallacoloo :  Done, done and done via https://github.com/tresf/lmms/commit/95f0b29bd0b311007f9d2360b27819be3413f6bd.  Also removed `configure`